bitwarden_license/src/Portal/Controllers/PoliciesController.cs

```diff
@@ -119,5 +122,37 @@ private IActionResult BuildPolicyView(Policy policy, PolicyType type)
                 return View(new PolicyEditModel(policy, _i18nService));
             }
         }
+
+        private async Task ValidateDependentPolicies(PolicyType type, Guid? orgId, bool enabled)
+        {
+            if (orgId == null)
+            {
+                throw new ArgumentNullException(nameof(orgId), "OrgId cannot be null");
+            }
+            
+            switch(type)
+            {
+                case PolicyType.MasterPassword:
+                case PolicyType.PasswordGenerator:
+                case PolicyType.TwoFactorAuthentication:
+                case PolicyType.OnlyOrg:
+                    break;
+                
+                case PolicyType.RequireSso:
+                    if (!enabled)
+                    {
+                        break;
+                    }
+                    var singleOrg = await _policyRepository.GetByOrganizationIdTypeAsync(orgId.Value, type);
+                    if (!singleOrg?.Enabled ?? false)
```

Comment:
This is pretty hairy, perhaps something like, `if (singleOrg?.Enabled != true)` is a bit more clear?

Reply:
Updated!